ote/src/clj/ote/components/db.clj

```diff
@@ -1,7 +1,6 @@
 (ns ote.components.db
   "Database connection pool component"
-  (:import (com.zaxxer.hikari HikariConfig HikariDataSource)
-           (org.postgresql.Driver))
+  (:import (com.zaxxer.hikari HikariConfig HikariDataSource))
```

Comment:
Just wodering why `org.postgresql.Driver` is removed, it's not needed?

Reply:
This form of one-word import form didn't have any effect, it was actually forbidden syntax according to spec but older Clojure versions didn't flag it.